### PR TITLE
SQLAlchemy GitHub actions

### PR DESCRIPTION
**Pull request by Sambigeara**

### CODE REVIEW

File: .github/workflows/prisma.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ on:
       - "prisma/**"
   push:
     tags:
-      - v*
+      - prisma/v*
```
Comment (Sambigeara) [inline .github/workflows/prisma.yaml:9]: @alexolivier just flagging this in case it somehow breaks your prisma publish flow?

**Review by charithe — APPROVED** [review]
LGTM.

It would be nice if the version is automatically derived from the tag because it's easy to forget to update `pyproject.toml`. On single project repositories [it is quite easy](https://github.com/cerbos/cerbos-sdk-python/blob/main/pyproject.toml#L23) but the prefixed tags in a monorepo probably makes it a bit more complicated 😞

**Comment by Sambigeara** [comment]
>It would be nice if the version is automatically derived from the tag because it's easy to forget to update `pyproject.toml`

Yeah, totally agree. Initial attempts to generate dynamically from scm have been fruitless (as you can see from the desperate commits above) but I'll continue to try tomorrow.

**Comment by Sambigeara** [comment]
Oooooh I think it's failing to retrieve the scm tag because `pyproject.toml` isn't in the git repo root. Something to look into tomorrow

**Pull request merged**